ci-operator/step-registry/openshift-qe/run-api-apf-customized-flowcontrol/openshift-qe-run-api-apf-customized-flowcontrol-commands.sh

```diff
@@ -238,8 +237,7 @@ function check_no_errors() {
   if [[ $error_count -le 0 ]]; then
     echo -e "Expected: No error logs found"
   else
-    echo -e "Errors found. Priority and Fairness settings did not properly catch the requests."
-    exit 1
+    echo -e "Errors found. Please double check if previous testing pod is deleted, those are expected error of APF"
```

Comment:
If the previous testing pods are not deleted, we may also see this error as in the test results I had put in the previous comment, right?

Reply:
@qiliRedHat The error message has been update, please review again.